src/modules/roc_pipeline/config.h

```diff
@@ -199,13 +206,20 @@ struct ReceiverConfig {
 
     //! Parameters common for all sessions.
     ReceiverCommonConfig common;
+
+    //! To specify which resampling algorithm will be used.
+    audio::ResamplerBackend resampler_backend;
```

Comment:
We already have `resampler_backend` in ReceiverSessionConfig. Instead duplicating it here and then copying to ReceiverSessionConfig, I suggest to remove this field and use `default_session.resampler_backend` instead.